frontend/src/v5/store/drawings/revisions/drawingRevisions.selectors.ts

```diff
@@ -18,26 +18,42 @@
 import { createSelector } from 'reselect';
 import { prepareRevisionData } from './drawingRevisions.helpers';
 import { IDrawingRevisionsState } from './drawingRevisions.redux';
-import { selectDrawingById } from '../drawings.selectors';
 
 const selectRevisionsDomain = (state): IDrawingRevisionsState => state.drawingRevisions;
 const selectDrawingIdParam = (_, drawingId: string) => drawingId;
 
-const selectRevisionsByDrawing = createSelector(
+export const selectRevisionsByDrawing = createSelector(
 	selectRevisionsDomain,
 	(state) => state.revisionsByDrawing || {},
 );
 
 export const selectRevisions = createSelector(
 	selectRevisionsByDrawing,
 	selectDrawingIdParam,
-	(revisionsByDrawing, drawingId) => revisionsByDrawing[drawingId]?.map((revision) => prepareRevisionData(revision)) || [],
+	(revisionsByDrawing, drawingId) => {
+		const revisions = (revisionsByDrawing[drawingId]?.map((revision) => prepareRevisionData(revision)) || []);
+		revisions.sort((a, b) => b.timestamp.getTime() - a.timestamp.getTime());
+		return revisions;
+	},
 );
 
-export const selectRevisionsPending = createSelector(
+export const selectActiveRevisions = createSelector(
 	selectRevisions,
-	selectDrawingById,
-	(revisions, drawing) => revisions.length !== drawing.revisionsCount,
+	(revisions) => revisions.filter((val) => !val.void),
+); 
+
+export const selectLastRevision = createSelector(
```

Comment:
Can this be renamed to `selectLatestRevision<Name>` or something similar as it doesn't actually return the revision itself, but its name/code?